README.md

```diff
@@ -51,7 +62,31 @@ You most likely may override `image`, `applicationPort` and `environment` for yo
 | `livenessTimeout` | Liveness probe timeout (seconds) | `3` |
 | `livenessPeriod` | Liveness probe period (seconds) | `15` |
 | `livenessFailureThreshold`| Liveness failure threshold | `3` |
-
+| `keyVaults`| This section is about adding the keyvault secrets to the file system see [Adding Azure Key Vault Secrets]()| none |
+
+## Adding Azure Key Vault Secrets
+We now support the integration of the key vault secrets into the docker filesystem so they don't have to be stored as _secrets_ (via side car or configMap) or 
```

Comment:
I would discourage docs that state we now support, that sort of thing get's stale quickly.

The features should be in there, CHANGELOG and commit messages can mention things like that, does that make sense

```suggestion
Key vault secrets are mounted to the container filesystem using what's called a [flexvolume](https://github.com/Azure/kubernetes-keyvault-flexvol)
```